ddl/db_test.go

```diff
@@ -2163,13 +2163,17 @@ func newTestMaxTableRowIDContext(c *C, d ddl.DDL, tbl table.Table) *testMaxTable
 	}
 }
 
+type tableMaxRowID interface {
```

Comment:
Why make it a seperate interface?

Reply:
Need comment.

Reply:
If we need it, please put it to the file of `ddl_test.go`.